packages/teleport-project-generator-react-basic/src/index.ts

```diff
@@ -36,6 +36,9 @@ export const createReactBasicGenerator = () => {
       generator: routingComponentGenerator,
       path: ['src'],
       fileName: 'index',
+      metaDataOptions: {
+        flavour: 'react',
```

Comment:
can this be the default and not be specified?

Reply:
Yeah, but I felt, in furture, if we plan to use the same routing plugin with one more flavor then we don't need to do all checks and then placing react as a fallback